GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -43,6 +43,13 @@ typedef void (^GIDVerifyCompletion)(GIDVerifiedAccountDetailResult *_Nullable ve
 /// The active configuration for this instance of `GIDVerifyAccountDetail`.
 @property(nonatomic, nullable) GIDConfiguration *configuration;
 
+/// Initialize a `GIDVerifyAccountDetail` object by specifying all available properties.
+///
+/// @param config The configuration to be used.
+/// @return An initialized `GIDVerifyAccountDetail` instance.
```

Comment:
This should say something about how and why the initializer will return nil.

Reply:
Added.

Reply:
I don't think this initializer should return nil. It's the parameter-less `init` that can return nil if we cannot make a `GIDConfiguration` automatically from the main bundle. 

Sorry for the churn here!

Reply:
Please update the nullability annotation and documentation here.

Reply:
Done.

Reply:
Done.